pkg/ccl/crosscluster/physical/stream_ingestion_job_test.go

```diff
@@ -56,10 +56,10 @@ func TestTenantStreamingCreationErrors(t *testing.T) {
 
 	telemetry.GetFeatureCounts(telemetry.Raw, telemetry.ResetCounts)
 	t.Run("destination cannot be system tenant", func(t *testing.T) {
-		sysSQL.ExpectErr(t, `pq: the destination tenant "system" \(0\) cannot be the system tenant`,
-			"CREATE TENANT system FROM REPLICATION OF source ON $1", srcPgURL.String())
+		sysSQL.ExpectErr(t, `pq: the destination tenant "" \(1\) cannot be the system tenant`,
+			"CREATE TENANT [1] FROM REPLICATION OF source ON $1", srcPgURL.String())
```

Comment:
can we keep the name version here and test both -- and that the name one keeps the name in the error?